data/api.toml

```diff
@@ -0,0 +1,56 @@
+enabled=true
```

Comment:
Why duplicate some config files? You could mount them directly from the already existing config folder. If any config needs changing it would now need to be changed in multiple places.

Reply:
This is due to disabling host networking, bc the config files are pointing to 127.0.0.1 for redis and mongo. I couldn't find a reasonable way to modify the original configs without making them docker specific with hostnames like mongo and redis.

Reply:
It may be better to add those specific things as `CMD_ARGS`. That would be a useful addition to the example docker-compose, and you would not need to duplicate the configuration files.

https://docs.wildduck.email/#/in-depth/docker

Reply:
ahh, I literally just saw the cmg args in the docs now, totally reinvented the wheel, updating, thanks!